core/state_transition.go

```diff
@@ -67,7 +67,7 @@ func (result *ExecutionResult) Revert() []byte {
 }
 
 // IntrinsicGas computes the 'intrinsic gas' for a message with the given data.
-func IntrinsicGas(data []byte, accessList types.AccessList, isContractCreation bool, isHomestead, isEIP2028 bool, isEIP3860 bool) (uint64, error) {
+func IntrinsicGas(data []byte, accessList types.AccessList, isContractCreation, isHomestead, isEIP2028, isEIP3860 bool) (uint64, error) {
```

Comment:
I would prefer the isContractCreation reverted. That's a completely different parameter than the others, which check for hard forks.

Reply:
Makes sense. Changed in [cca3909](https://github.com/ethereum/go-ethereum/pull/29036/commits/cca39093c1fbf688fd190c9c73e8587ec2b8dfe9).